### ISSUE TITLE
[Bug]: Navigation buttons (back/forward) are disabled

### ISSUE THREAD

**Issue by skynbe** [th-ch/youtube-music#1762]
### Preflight Checklist

- [X] I use the latest version of YouTube Music (Application).
- [X] I have searched the [issue tracker](https://github.com/th-ch/youtube-music/issues) for a bug report that matches the one I want to file, without success.
- [X] I understand that **th-ch/youtube-music has NO affiliation with Google or YouTube**

### YouTube Music (Application) Version

3.2.2

### Checklists

- [ ] I use the portable version of the YouTube Music Application.
- [ ] I can reproduce this issue in the [official YTM web version](https://music.youtube.com).

### What operating system are you using?

macOS

### Operating System Version

mac 12.3.1

### What CPU architecture are you using?

arm64 (including Apple Silicon)

### Reproduction steps

Navigation buttons (in the right upper side) are visible, but disabled. 
([Navigation > Go back] or [Navigation > Go forward] works well.)

### Expected Behavior

Navigation buttons are enabled

### Actual Behavior

Navigation buttons (in the right upper side) are visible, but disabled. 
([Navigation > Go back] or [Navigation > Go forward] works well.)

### Enabled plugins

Navigation

**Comment by JellyBrick** [comment]
I think it's a duplicate of #1381
Can you reproduce this issue with [these reproduction steps](https://github.com/th-ch/youtube-music/issues/1381#issuecomment-1939750899)?

**Comment by skynbe** [comment]
@JellyBrick Yes, because navigation buttons are always disabled (even at the home screen).
